src/rp2_common/hardware_irq/include/hardware/irq.h

```diff
@@ -289,6 +289,65 @@ void irq_set_pending(uint num);
  * \note This is an internal method and user should generally not call it.
  */
 void irq_init_priorities(void);
+
+/*! \brief Claim ownership of a user IRQ on the calling core
+ *  \ingroup hardware_irq
+ *  
+ * User IRQs are numbered 26-31 and are not connected to any hardware, but can be triggered by \ref irq_set_pending.
+ *
+ * \note User IRQs are entirely core local.
```

Comment:
Does "entirely core local" mean that each core has it's own independent set of User IRQs? Might be worth being slightly more verbose? :shrug: